platform/api/equinixmetal/api.go

```diff
@@ -362,6 +362,7 @@ ExecStart=/usr/bin/curl --retry-delay 1 --retry 120 --retry-connrefused --retry-
 # We don't verify signatures because the iPXE script isn't verified either
 # (and, in fact, is transferred over HTTP)
 
+ExecStartPre=-/bin/bash -c 'shopt -s nullglob; for disk in /dev/*da? /dev/nvme?n1; do wipefs --all --force ${disk}; done'
```

Comment:
In case LVM volumes are left over, it also makes sense to run `lvchange -an /dev/mapper/*` before `flatcar-install`.

Reply:
Also, it must be `$${disk}` or it will be substituted as argument for bash itself instead of being evaluated inside the shell

Reply:
I'll file a PR

Reply:
https://github.com/flatcar-linux/mantle/pull/305